presto-native-execution/presto_cpp/main/types/tests/PlanConverterTest.cpp

```diff
@@ -162,3 +162,7 @@ TEST_F(PlanConverterTest, scanAggBatch) {
   ASSERT_NE(partitionAndSerializeNode, nullptr);
   ASSERT_EQ(partitionAndSerializeNode->numPartitions(), 3);
 }
+
+TEST_F(PlanConverterTest, windowNodeCoredump) {
+  assertToVeloxQueryPlan("WindowNodeCoredump.json");
```

Comment:
Tests that use hard-coded JSON files are difficult to understand and maintain. Would it be possible to add a simple TPC-H-based query that manifest the issue to presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java ?